django_hosts/test/testcases.py

```diff
@@ -0,0 +1,20 @@
+from django.test import SimpleTestCase, TestCase, TransactionTestCase
+
+from .client import HostsClient, AsyncHostsClient
+
+
+class HostsTestCaseMixin:
+    client_class = HostsClient
+    async_client_class = AsyncHostsClient
+
+
+class SimpleHostsTestCase(HostsTestCaseMixin, SimpleTestCase):
+    pass
+
+
+class TransactionHostsTestCase(HostsTestCaseMixin, TransactionTestCase):
+    pass
+
+
+class HostsTestCase(HostsTestCaseMixin, TestCase):
```

Comment:
I chose to give distinct names to the clients and testcases as the ones from Django, including `Hosts` in their name, but perhaps it may be easier to adopt if it's called just `TestCase`, which means people just need to update their imports?

```diff
- from django.test import TestCase
+ from django_hosts.test import TestCase
```

Granted they could `import HostsTestCase as TestCase`, but it feels like it might be a common use case... Also, unlikely to need both the Django test case and hosts test in the same file, so shouldn't cause any conflict (and if it does, users can import as).